gatherers/DockerComposeGatherer.go

```diff
@@ -0,0 +1,70 @@
+package gatherers
+
+import (
+	"github.com/bomoko/lagoon-facts/utils"
+	"log"
+)
+
+type dockerComposeGatherer struct {
+	GatheredFacts []GatheredFact
+}
+
+type dockerComposeService struct {
+	Name string
+	Description string
+	Image string
+	Labels map[interface {}]interface {}
+	Build map[interface {}]interface {}
+}
+
+func (p *dockerComposeGatherer) AppliesToEnvironment() bool {
+	applies := false
+
+	lagoonConfigBytestream, err := LoadDockerComposeConfig("docker-compose.yml")
+	if err != nil {
+		log.Printf("Couldn't load docker-compose.yml file: ", err.Error())
+	}
+
+	config, err := utils.UnmarshallYamlToStructure(lagoonConfigBytestream)
+	if err != nil {
+		log.Fatalf("There was an issue unmarshalling the docker-compose.yml file: %s", err)
+	}
+
+	if config.Services != nil {
+		applies = true
+
+		for k, element := range config.Services {
```

Comment:
important to note that any service with a lagoon.type=none will not be in Lagoon, but will still show up here - maybe we need to think about whether this is the right place to collect services from.

Reply:
I'm thinking the docker-compose gatherer itself might not be needed